packages/gallery/src/components/item/itemView.js

```diff
@@ -789,7 +789,7 @@ class ItemView extends React.Component {
         label = '';
         break;
       case 'text':
-        label = 'Text item';
+        label = extractTextItemContent(this.props.html);
```

Comment:
@yuvalab 
Can you change the entire switch case to an object? Same as we did before?